tests/NgTests/Infrastructure/MemoryStorage.cs

```diff
@@ -46,7 +46,8 @@ protected override Task OnSave(Uri resourceUri, StorageContent content, Cancella
         protected override Task<StorageContent> OnLoad(Uri resourceUri, CancellationToken cancellationToken)
         {
             StorageContent content;
-            Content.TryGetValue(resourceUri, out content);
+            // Ignore query strings
```

Comment:
This comment is kinda confusing without the context that it used to be just `resourceUri` instead of `new Uri(resourceUri.GetLeftPart(UriPartial.Path))`. I think this segment would be much clearer if you had it as...

```
var resourceUriWithoutQuery = new Uri(resourceUri.GetLeftPart(UriPartial.Path));
Content.TryGetValue(resourceUriWithoutQuery, out content);
```